montepy/data_inputs/fill.py

```diff
@@ -525,9 +525,11 @@ def _value_node_generator():
                         payload.append(self.universes[i][j][k].number)
         else:
             payload = [
-                self.universe.number
-                if self.universe is not None
-                else self.old_universe_number
+                (
```

Comment:
Are these parens necessary?

Reply:
That's added by `black` so I'm not messing with it. I think it does make it more legible though expands it.